src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleTypeNameConverter.java

```diff
@@ -26,6 +26,7 @@
   /** A map from primitive types in proto to NodeJS counterparts. */
   private static final ImmutableMap<Field.Kind, String> PRIMITIVE_TYPE_MAP =
```

Comment:
Got it - I'd like to come back to this issue in another PR. I'll file an issue to track.